templates/blackbox.yml.j2

```diff
@@ -0,0 +1,4 @@
+modules:
```

Comment:
Aware this is a template not a file - will be looking to add more templation in here over time, but for now it can just be a http 200 checker

Reply:
(if you want users and passwords on a authenticated endpoint, for instance, you have to define them in here rather than in the prometheus.yaml file)

Reply:
Okay :+1: